docs/bokeh/Makefile

```diff
@@ -26,5 +26,7 @@ html:
 	&& echo \
 	&& echo "Build finished in $$(($$(date +%s)-start)) seconds. The HTML pages are in $(BUILDDIR)/html."
 
+test-accessibility:
+	python tests/test_docs_accessibility.py
```

Comment:
Generally speaking, everything under `tests` is something runnable by `pytest` and is expected to pass/fail based on some criteria. Since this is not the case with the current `test_docs_accessibility.py` script, I'd suggest moving it  to a different location. Some ideas:

* somewhere under the `scripts` dir
* somewhere under the `docs/bokeh` dir

I'd also suggest renaming it to `docs_accessibility_audit.py` to reflect that it's not actually a test, but more of a report generator (later on, perhaps an actual pass/fail test might call this script as a helper to do the work)